pkg/logcli/seriesquery/series.go

```diff
@@ -11,24 +14,71 @@ import (
 
 // SeriesQuery contains all necessary fields to execute label queries and print out the results
 type SeriesQuery struct {
-	Matchers []string
-	Start    time.Time
-	End      time.Time
-	Quiet    bool
+	Matcher       string
```

Comment:
This should take a slice of matcher clauses, see https://grafana.com/docs/loki/latest/api/#series

Reply:
i debated this but honestly I couldn't figure out why that would be useful? I couldn't think of a usecase for this from logcli? I may totally be missing one though.

Reply:
It's probably ok. I think this is more important for Prometheus which can't combine multiple metric names in the same matcher clause, but loki doesn't have this problem. 

Anyways, we can always add it in the future if there's a case where we need it. LGTM